current_docker.opam

```
@@ -24,6 +24,7 @@ depends: [
   "ppx_deriving"
   "lwt"
   "dockerfile"
+  "ocaml-version" {>="2.5.0"}
```

Should this be 2.6.0?
```suggestion
  "ocaml-version" {>="2.6.0"}
```